doc/news/changes/minor/20240820Ghadban

```diff
@@ -0,0 +1,4 @@
+New: FESystem::compare_for_domination() now accepts two FESystem objects
+with different number of base elements
```

Comment:
Let's add a period here at the end to make it a complete sentence :-)

Reply:
Done!